python/tank/deploy/tank_commands/update.py

```diff
@@ -109,14 +146,39 @@ def run_interactive(self, log, args):
             log.info("")
             
             if console_utils.ask_yn_question("Continue with full update?"):
-                check_for_updates(log, self.tk)
+                check_for_updates(log, 
+                                  self.tk,
+                                  env_name=None,
+                                  engine_instance_name=None, 
+                                  app_instance_name=None)
             
             return
         
         env_filter = None
         engine_filter = None
         app_filter = None        
+        external_path = None
         
+        # look for an --external argument
+        for arg in args:
+            if arg.startswith("--external="):
+                # remove it from args list
+                args.remove(arg)                                
```

Comment:
yeah it's getting a bit too much now. in the next change we make, let's fix it! :)